rudder-web/src/main/scala/bootstrap/liftweb/AppConfigAuth.scala

```diff
@@ -166,6 +170,58 @@ class AppConfigAuth extends Loggable {
 
 }
 
+/*
+ * A trivial extension to the SimpleUrlAuthenticationFailureHandler that correctly log failure
+ */
+class RudderUrlAuthenticationFailureHandler(failureUrl: String) extends SimpleUrlAuthenticationFailureHandler(failureUrl) {
+  override def onAuthenticationFailure(request: HttpServletRequest, response: HttpServletResponse, exception: AuthenticationException): Unit = {
+    LogFailedLogin.warn(exception, request)
+    super.onAuthenticationFailure(request, response, exception)
+  }
+}
+
+object LogFailedLogin {
+
+  def warn(ex: AuthenticationException, request: HttpServletRequest): Unit = {
+    ApplicationLogger.warn(s"Login authentication failed for user '${getUser(ex)}' from IP '${getRemoteAddr(request)}': ${ex.getMessage}")
```

Comment:
Ha - i see, this is because is it redirected, so it is indeed non interesting.
Does the getQueryString returns something useful ?